test/e2e/apimachinery/resource_quota.go

```diff
@@ -91,6 +90,7 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		ginkgo.By("Creating a ResourceQuota")
 		quotaName := "test-quota"
 		resourceQuota := newTestResourceQuota(quotaName)
+		resourceQuota.Spec.Hard[v1.ResourceServices] = resource.MustParse("2")
```

Comment:
What's the reason for this change?

Reply:
Check this [failure case](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/100412/pull-kubernetes-e2e-kind-ipv6/1373025220083847168) in PR history. This test created two services and expects the third service creation to be failed due to the quota issue. However it didn't fail so I set the quota to 2, then it worked. It was 10 originally by default.

Reply:
@pacoxu you fixed the bug related to this https://github.com/kubernetes/kubernetes/commit/15867d9e8a1faf007f6df563c26a9b5e8744b2a1#diff-e1a46008c4ddbb9f83858cb22ce56ca0f820b78142a1e8abbbb11aa59127a09b , can you PTAL

Reply:
do we have resolution on this?  I do not understand this change

Reply:
Hi @thockin, you can see that as we initialize the quota using `newTestResourceQuota`, in line 1561, by default we can create 10 services:
https://github.com/kubernetes/kubernetes/blob/49a60be8ab5578097dc2dd2b851213d9f8ca1105/test/e2e/apimachinery/resource_quota.go#L1557-L1578
And read through the test, it basically expect the first 2 service (one of type cluster IP and the other of type node port) to be created successfully, and expect the third creation (which happens to be a service of LB type) to fail. 

What I remember when I did the tests months back was, before this PR, the third service(the LB type one) creation failed, but it's because the `ServiceLBNodePortControl` feature was not turned on by default. It has nothing to do with quota here. And since this PR enabled that feature by default, the third service was successfully created which failed the test. So it's a bug in the test case and all I should do is just set the quota to the correct value which is 2.